website/docs/r/catalog.html.markdown

```diff
@@ -58,8 +58,8 @@ The following arguments are supported:
 * `description` - (Optional) Description of catalog
 * `storage_profile_id` - (Optional, *v3.1+*) Allows to set specific storage profile to be used for catalog. **Note.** Data
 source [vcd_storage_profile](/providers/vmware/vcd/latest/docs/data-sources/storage_profile) can help to lookup storage profile ID.
-* `delete_recursive` - (Required) When destroying use delete_recursive=True to remove the catalog and any objects it contains that are in a state that normally allows removal
-* `delete_force` - (Required) When destroying use `delete_force=true` with `delete_recursive=true` to remove a catalog and any objects it contains, regardless of their state
+* `delete_recursive` - (Optional, but recommended) When destroying use delete_recursive=True to remove the catalog and any objects it contains that are in a state that normally allows removal
```

Comment:
Should we state that they're now false if not set? (as they're optional)

PS: Same doubt for org, org_vdc.

Reply:
Thanks. Added in all three resources